tests/integration/test.conflicts.js

```diff
@@ -68,6 +68,12 @@ adapters.forEach(function (adapter) {
     });
 
     it('#2882/#2883 last_seq for empty db', function () {
+      // CouchDB 2.0 sequence numbers are not
+      // incremental so skip this test
+      if (testUtils.isCouchMaster()) {
+        return true;
+      }
+      
```

Comment:
This line has trailing whitespace, its not a big deal but very handy to configure your editor to make sure its trimmed, cheers